### ISSUE TITLE
Push notifications

### ISSUE THREAD

**Issue by auser** [fullstackreact/react-native-firestack#22]
[Cloud messaging](https://firebase.google.com/docs/cloud-messaging/) 

Add cloud messaging.

**Comment by saleeh93** [comment]
did you stated anything yet..?
I am happy to contribute

**Comment by auser** [comment]
@saleeh93 I have not started this feature yet. Contributions would be great here!

I'll hang out in the [gitter](https://gitter.im/fullstackreact/react-native-firestack) room if you want to discuss.

**Comment by saleeh93** [comment]
@auser lets catch up there

**Comment by auser** [comment]
@saleeh93 sounds good

**Comment by auser** [comment]
@saleeh93 the PR #30 has the introduction of the JS/Android version, if you wanted to start on the iOS version.

**Comment by auser** [comment]
@saleeh93 @advans I just pushed a branch `feature/messaging` that includes stubs for iOS using the same JS api, if either of you want to/get a chance to update the messaging iOS side from: https://firebase.google.com/docs/cloud-messaging/ios/client

**Comment by saleeh93** [comment]
@auser Little busy right now

**Comment by auser** [comment]
@saleeh93 no problemo. Just keeping you updated on progress :)

**Comment by moooji** [comment]
It looks like most of the work has already been done here!
https://github.com/evollu/react-native-fcm

Maybe there is a possibility to merge the libraries?

**Comment by auser** [comment]
Yes, I have reached out to the author, but he's pretty resistant to merge the two libraries, as far as I can tell. I'll reach out to him again and see his interest. Feel free to reach out to him as well. Perhaps we make an issue?

**Comment by moooji** [comment]
I have opened an issue here https://github.com/evollu/react-native-fcm/issues/177. Feel free to comment and add your thoughts.

**Comment by arthurvi** [comment]
For now we should just use evollu/react-native-fcm if we want to implement push notifications?

**Comment by ezos86** [comment]
Waiting for this answer as well.
